staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go

```diff
@@ -178,6 +178,7 @@ func (c completedConfig) NewWithDelegate(delegationTarget genericapiserver.Deleg
 		endpointsLister: s.endpointsLister,
 	}
 	s.GenericAPIServer.HandlerContainer.Handle("/apis", apisHandler)
+	s.GenericAPIServer.HandlerContainer.Handle("/apis/", apisHandler)
```

Comment:
this has the side-effect of making any subpath of `/apis/` (like `/apis/foo/bar/baz`) be handled by `apisHandler`. Is that what we want?